src/components/Modal.js

```diff
@@ -39,6 +40,12 @@ const Modal = (props) => {
         needsSafeAreaPadding,
     } = getModalStyles(props.type, useWindowDimensions());
 
+    //register escape key listener via effect
```

Comment:
It looks like the last style run timed out, but this should have a space between `//` and the comment, and start with a capital letter.